enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/felles/Vedtak.kt

```diff
@@ -37,6 +37,7 @@ enum class TilkjentYtelseStatus {
 
 enum class Vilkårsresultat(val beskrivelse: String) {
     OPPFYLT("Vilkåret er oppfylt når alle delvilkår er oppfylte"),
+    AUTOMATISK_OPPFYLT("Delvilkår er oppfylt med automatisk beregning"),
```

Comment:
Ble litt usikker på om dette er diskutert med noen (Johan / Mattis):

Mulige løsninger for "automatisk" vurdert:
1) Slik som du har lagt opp til her - eget resultat
2) En boolsk verdi `Automatisk vurdert` på `Vilkårsvurdering`/`Vurdering` (delvilkår) 


Jeg ser fordeler og ulemper med begge deler:
1) Eksplisitt og tydelig at dette er automatisk oppfylt og følgelig lett å resonnere rundt. Kommer til å kreve en nøye gjennomgang av all logikk rundt bruk av vilkårsresultat for å sørge for at det ikke blir kluss i "oppfylt"-sjekkene som ligger rundt omkring
2) Krever ikke en omskriving av eksisterende kode for å verifisere hva som er oppfylt/ikke-oppfylt, men blir kanskje litt gjemt/skjult og vanskelig å lese ut entydig at dette er automatisk vurdert. Litt knotete at to objekter trenger verdien satt? Må kanskje leve med "null"=>"false" til vi evt. patcher gamle delvilkårsvurderinger